app/celery/scheduled_tasks.py

```diff
@@ -418,12 +418,13 @@ def zendesk_new_email_branding_report():
         brands_with_no_organisation=brands_with_no_organisation,
     )
 
-    ticket = NotifySupportTicket(
-        subject="Review new email brandings",
-        message=message,
-        ticket_type=NotifySupportTicket.TYPE_TASK,
-        technical_ticket=False,
-        ticket_categories=["notify_no_ticket_category"],
-        message_as_html=True,
-    )
-    zendesk_client.send_ticket_to_zendesk(ticket)
+    if current_app.config["NOTIFY_ENVIRONMENT"] in ["live", "production", "test"]:
```

Comment:
optional: we have `["live", "production", "test"]` repeated like 6 times in this repo now. Maybe we could have a single source of truth for this, like a `PROD_OR_TEST` or something, set in init file or in utils?